packages/node/src/integrations/utils/http.ts

```diff
@@ -221,3 +204,20 @@ export function normalizeRequestArgs(
     return [requestOptions];
   }
 }
+
+function parseRequestOptions(requestOptions: RequestOptions): {
+  protocol: string;
+  hostname: string;
+  port: string;
+} {
+  const protocol = requestOptions.protocol || '';
+  const hostname = requestOptions.hostname || requestOptions.host || '';
+  // Don't log standard :80 (http) and :443 (https) ports to reduce the noise
+  // Also don't add port if the hostname already includes a port
```

Comment:
Also noticed that `host` may include the port (e.g. `localhost:3000`), so we better check that this is not the case before re-adding the port.